src/config.c

```diff
@@ -2669,6 +2677,7 @@ standardConfig configs[] = {
 
     /* Unsigned int configs */
     createUIntConfig("maxclients", NULL, MODIFIABLE_CONFIG, 1, UINT_MAX, server.maxclients, 10000, INTEGER_CONFIG, NULL, updateMaxclients),
+    createUIntConfig("unixsocketperm", NULL, IMMUTABLE_CONFIG, 0, 0777, server.unixsocketperm, CONFIG_DEFAULT_UNIX_SOCKET_PERM, OCTAL_CONFIG, NULL, NULL),
```

Comment:
Same comment as the other two PRs, let's moved the default value here and remove the unnecessary abstraction.